frontend/src/components/readsets/ReadsetsListContent.tsx

```diff
@@ -1,105 +1,345 @@
+            switch (readsetStatus.new) {
+                case ReleaseStatus.RELEASED:
+                    newReleaseCount++
+                    break
+                case ReleaseStatus.BLOCKED:
+                    newBlockCount++
+                    break
+                case ReleaseStatus.AVAILABLE:
+                    newAvailableCount++
+                    break
             }
         }
-    }
+        const popconfirmTitle = [
+            `From the selected dataset readsets:`,
+            newReleaseCount > 0 ? `${newReleaseCount} will be released` : '',
+            newBlockCount > 0 ? `${newBlockCount} will be blocked` : '',
+            newAvailableCount > 0 ? `${newAvailableCount} will be made available` : '',
+            `Do you want to save these changes?`
+        ].map((str, index) => <div key={index}>{str}</div>)
+    
+        return <div>
+            <Button
+                style={{ margin: 6 }}
+                onClick={() => {
+                    releaseStatusManager.setAllReleaseStatus(ReleaseStatus.RELEASED)
+                }}
+                disabled={!releaseAllEnabled}>
+                Release All
+            </Button>
+            <Button
+                style={{ margin: 6 }}
+                onClick={() => {
+                    releaseStatusManager.setAllReleaseStatus(ReleaseStatus.BLOCKED)
+                }}
+                disabled={!blockAllEnabled}>
+                Block All
+            </Button>
+            {isAdmin && <Button
+                    style={{ margin: 6 }}
+                    onClick={() => {
+                        releaseStatusManager.setAllReleaseStatus(ReleaseStatus.AVAILABLE)
+                    }}
+                    disabled={!undoAllReleaseAndBlockEnabled}>
+                    Undo All Release/Block
+                </Button>}
+            <Button
+                style={{ margin: 6 }}
+                onClick={() => {
+                    releaseStatusManager.undoChanges()
+                }}
+                disabled={!undoChangesEnabled}>
+                Undo Changes
+            </Button>
+            { !saveChangesEnabled && canUpdateReleaseStatus && !isAdmin
+            ? <Popover content={"All dataset readsets need to be released or blocked to save changes."}>
+                <Button
+                style={{ margin: 6 }}
+                type={"primary"}
+                disabled={!saveChangesEnabled}>
+                    Save Changes
+                </Button>
+            </Popover>
+            : <Popconfirm
+                title={popconfirmTitle}
+                onConfirm={async () => {
+                    await releaseStatusManager.updateReleaseStatus()
+                    await refreshDataset()
+                }}
+                disabled={!saveChangesEnabled}
+                icon={null}
+            >
+                <Button
+                style={{ margin: 6 }}
+                type={"primary"}
```

Comment:
the save changes doesn't seem to be working, the app updates to the chosen values but it automatically returns to its original value before the update

Reply:
The lack of positive feedback might be missing. On my side, on another computer, it works.